app/views/NewServerView.js

```
@@ -201,6 +201,11 @@ class NewServerView extends React.Component {
 			url = parsedUrl.host;
 		}
 
+		// host never can contain a ':'
```

We should return an error, actually.
So the user can know what was wrong.